lang/cs/Org.Apache.REEF.Network/Group/Task/Impl/CommunicationGroupNetworkObserver.cs

```diff
@@ -109,7 +109,7 @@ private IObserver<GroupCommunicationMessage> GetOperatorHandler(string operatorN
             {
                 if (!_handlers.ContainsKey(operatorName))
                 {
-                    LOGGER.Log(Level.Info, "handler for operator {0} has not been registered." + operatorName);
+                    LOGGER.Log(Level.Warning, "Handler for operator {0} has not been registered. You should never reach here!!!" + operatorName);
```

Comment:
With the observer registered in the constructor, this scenario should not happen. If happens, it is error. I was to remove the retry later. Now I just removed it and throw exception in this case.